### PR TITLE
Create aditro-submit-1.md

### PR DESCRIPTION
**Pull request by rianaditro**

### CODE REVIEW

File: aditro-submit-1.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Name: Rian
+Email: rianaditro@gmail.com.id
```
Comment (rian669) [inline aditro-submit-1.md]: email is not valid

File: aditro-submit-1.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+Status: very handsome
```
Comment (rian669) [inline aditro-submit-1.md:3]: this data is not needed

**Review by rian669 — CHANGES_REQUESTED** [review]
fix it

File: aditro-submit-1.md (hunk 2 above)
Comment (rianaditro) [reply aditro-submit-1.md:3]: what it should be?

File: aditro-submit-1.md (hunk 1 above)
Comment (rianaditro) [reply aditro-submit-1.md]: ok. fixed it

**Review by rianaditro — COMMENTED** [review]
updated

**Comment by rian669** [comment]
ok

**Review by rian669 — COMMENTED** [review]
ok

**Pull request closed without merging**